interactive_engine/executor/assembly/groot/src/store/backup.rs

```diff
@@ -44,13 +45,15 @@ pub extern "C" fn openGraphBackupEngine(
 #[no_mangle]
 pub extern "C" fn closeGraphBackupEngine(handle: GraphBackupHandle) {
     let ptr = handle as *mut GraphBackupEngine;
+    trace!("closeGraphBackupEngine");
     unsafe {
-        Box::from_raw(ptr);
+        drop(Box::from_raw(ptr));
```

Comment:
Can drop this unnecessary code

Reply:
Similarly, if a point already been wrapped into a Rust struct, e.g. Box, or CString, they will be dropped automatically

Reply:
There would be a warning like

```
warning: unused return value of `std::boxed::Box::<T>::from_raw` that must be used
   --> /Users/runner/work/GraphScope/GraphScope/interactive_engine/executor/engine/pegasus/pegasus/src/communication/buffer.rs:121:13
    |
121 |             Box::from_raw(ptr.as_ptr());
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: call `drop(Box::from_raw(ptr))` if you intend to drop the `Box`
    = note: `#[warn(unused_must_use)]` on by default
help: use `let _ = ...` to ignore the resulting value
    |
121 |             let _ = Box::from_raw(ptr.as_ptr());
    |             +++++++

```